src/ClassicUO.Client/LegionScripting/doc.md

```
@@ -0,0 +1,2079 @@
+<details>
+<summary><h3>Pathfind(x, y, z, distance)</h3></summary>
+
+ Attempt to pathfind to a location.  This will fail with large distances.  
+ Example:
+ ```py
+ API.Pathfind(1414, 1515)
+ ```
+
+
+**Parameters**  
+| Name | Type | Optional | Description |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)